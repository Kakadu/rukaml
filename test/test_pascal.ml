let () = print_endline "hello"

let () =
  let open AST in
  let prog = [], [ Assgn ("x", EVar "x") ] in
  print_endline @@ show_program prog
;;
