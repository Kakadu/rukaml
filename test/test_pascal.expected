hello
([], [(Assgn ("x", (EVar "x")))])
